utils/frame/try-runtime/cli/src/lib.rs

```diff
@@ -93,6 +151,10 @@ impl TryRuntimeCmd {
 	pub async fn run<B, ExecDispatch>(&self, config: Configuration) -> sc_cli::Result<()>
 	where
 		B: BlockT,
+		B::Hash: FromStr,
+		<B::Hash as FromStr>::Err: std::fmt::Debug,
```

Comment:
`Debug` is already in scope.
```suggestion
		<B::Hash as FromStr>::Err: Debug,
```